sidechain/consensus/aura/src/lib.rs

```diff
@@ -535,9 +535,8 @@ mod tests {
 
 		// hard to compare actual numbers but we can at least ensure that the general concept works
 		assert!(
-			proposing_remaining_duration(&slot_info, duration_now()) > SLOT_DURATION / 2
-				&& proposing_remaining_duration(&slot_info, duration_now())
-					< SLOT_DURATION.mul_f32(BLOCK_PROPOSAL_SLOT_PORTION + 0.01)
+			proposing_remaining_duration(&slot_info, duration_now())
+				< SLOT_DURATION.mul_f32(BLOCK_PROPOSAL_SLOT_PORTION + 0.01)
```

Comment:
The first condition of this assertion `> SLOT_DURATION / 2` is only true if the `BLOCK_PROPOSAL_SLOT_PORTION` is `> 0.5`, which is not guaranteed (and so this test failed when I lowered the value to `0.4` for testing)